docs/content/direct/observability.md

```diff
@@ -4,20 +4,19 @@ KubeStellar provides endpoints and integrations for observability and monitoring
 
 ## Metrics Endpoints
 
-
 KubeStellar controllers expose Prometheus-compatible metrics endpoints. These endpoints respond to HTTP requests for metrics and can be queried by any monitoring system; KubeStellar does not mandate how metrics are collected or scraped. For an example of collecting both metrics and debug endpoint data using Prometheus, see the [monitoring](https://github.com/kubestellar/kubestellar/tree/main/monitoring/) directory. This is just one possible approach; KubeStellar does not require or mandate any specific monitoring tool or method.
 
 ### Metrics Endpoint Table
 
-| Controller                    | Protocol | Port  | Path     | AuthN/AuthZ | Notes |
-|-------------------------------|----------|-------|----------|-------------|-------|
-| kubestellar-controller-manager | HTTPS    | 8443  | /metrics | Kubernetes client authentication required when using the Service | Service: `kubestellar-controller-manager-metrics-service` (default port, configurable via Helm values). |
-| kubestellar-controller-manager | HTTPS    | 8443  | /metrics | None (direct pod access) | Access via Pod port 8443 (default, configurable via Helm values). |
-| ks-transport-controller       | HTTP     | 8090  | /metrics | None (in-cluster) | Default port, configurable via Helm values. |
-| status-addon-controller       | HTTP     | 9280  | /metrics | None (in-cluster) | Default port, configurable via Helm values. |
-| status-agent-controller       | HTTP     | 8080  | /metrics | None (in-cluster) | Default port, configurable via Helm values. |
+| Controller                     | Protocol | Port  | Path     | AuthN/AuthZ                                               | Notes                                                         |
+|-------------------------------|----------|-------|----------|-----------------------------------------------------------|---------------------------------------------------------------|
+| kubestellar-controller-manager | HTTPS    | 8443  | /metrics | Kubernetes client authentication required (Service or Pod) | Service: `kubestellar-controller-manager-metrics-service`.    |
+| kubestellar-controller-manager | HTTP     | 8080  | /metrics | None (debug; not recommended for production)               | Debug endpoint; typically disabled in production.              |
```

Comment:
I do not know anything about disabling this endpoint in production. How would that be done?
You can see at https://github.com/kubestellar/kubestellar/blob/33889de6609f91eb0c3e00a03bbafce61fd62ede/core-chart/templates/postcreatehooks/kubestellar-controller.yaml#L238 that the listening socket is bound to IP address 127.0.0.1. That means that this is reachable only (a) from within the Pod and (b) through the magic of `kubectl port-forward`.

Reply:
FYI, the ability to use port-forwardig IS subject to authorization --- even though this is not documented (https://github.com/kubernetes/website/issues/51700).